### ISSUE TITLE
Consistent cvar naming

### ISSUE THREAD

**Issue by aufau** [mvdevs/jk2mv#74]
Here are my propositions:

1. Rename all cvars to their correct prefix, leaving `mv_` for multiversion cvars (so eg. mv_allowhttpdownloads needs to change)
2. Same, but add `mv_` aliases for all new cvars 
3.  Rename all new cvars to `mv_`
4. Use `mv_` prefix only for important new features

Personally I'm not sure. It's very useful for users to have new cvars grouped together, but it's inconsistent with original naming scheme and can cause mistakes when coding. 2 is the best of both worlds, although number of new cvars is rising so it may get messy too. Aliasing implementation is trivial. At least all internal cvar variables should have correct prefix based on subsystem they reside in.

**Comment by ouned** [comment]
option 1

**Comment by Daggolin** [comment]
If I understand this proposal correct option 1 would mean that almost all current mv_ cvars would need to change. So anyone updating their servers or clients would have to redo all their previous mv_ setup, by trying to figure out what the cvar is called now (most likely just a new prefix).
But as people usually update their servers without redoing the whole config not everyone would notice and we would throw lots of servers into a config-chaos. And I assume clients wouldn't be too happy either.
So if you really want to rename them please go with option 2 and use aliases.  
  
I think option 3 is not a good option either, because many of the current cvars reflect the commonly known cvars around ioq3 or JKA. Changing for instance `con_height` or `r_dynamicGlow` to something starting with mv_ doesn't sound like a good option to me.  
  
I think using the mv_ prefix for some new features or options only related to jk2mv (option 4) might be a nice option. That or keeping aliases for the old mv_ cvars if we want to pay more attention to the old distinction between r_, cl_, sv_, ... (option 2).

**Comment by aufau** [comment]
Here are all mv_ cvars:

```
mv_allowDownload
mv_apiEnabled (?)
mv_blockchargejump
mv_blockspeedhack
mv_coloredTextShadows
mv_consoleShiftRequirement
mv_fixbrokenmodels
mv_fixforcecrash
mv_fixgalaking
mv_fixnamecrash
mv_fixsaberstealing
mv_fixturretcrash
mv_httpdownloads
mv_httpserverport
mv_nameShadows
mv_saberGlow
mv_serverversion
mv_slowrefresh
```

Doesn't look as bad as I thought. I'd just rename mv_slowrefresh, mv_saberGlow and mv_consoleShiftRequirement. Download cvars can stay as they are I think, and mv_coloredTextShadows is kinda multiversion feature.

**Comment by ouned** [comment]
we kinda decided to leave it how it is? Can it be closed?

**Comment by aufau** [comment]
I just want to rename mv_saberGlow to r_saberGlow. I'm also collecting these tiny cvar corrections (eg. default value changes) to push them in one commit.

**Comment by Daggolin** [comment]
I guess we can rename mv_saberGlow to r_saberGlow and close this issue?

**Comment by aufau** [comment]
Closed by ab6f7bbb045d70af1a448580ef8d117aabfc83df
